pkgs/top-level/all-packages.nix

```diff
@@ -12161,11 +12161,12 @@ with pkgs;
     };
   };
 
+  # linux mptcp is based on the 4.4 kernel
   linux_mptcp = callPackage ../os-specific/linux/kernel/linux-mptcp.nix {
     kernelPatches =
       [ kernelPatches.bridge_stp_helper
         kernelPatches.p9_fixes
-        kernelPatches.DCCP_double_free_vulnerability_CVE-2017-6074
```

Comment:
Could you remove the DCCP kernel patch, it is not referenced anywhere else.

Reply:
it is already removed from pkgs/os-specific/linux/kernel/patches.nix I believe

Reply:
Doesn't seem so:
```
$ rg DCCP_double_free
pkgs/top-level/all-packages.nix
12402:        kernelPatches.DCCP_double_free_vulnerability_CVE-2017-6074

pkgs/os-specific/linux/kernel/patches.nix
61:  DCCP_double_free_vulnerability_CVE-2017-6074 = rec
62:    { name = "DCCP_double_free_vulnerability_CVE-2017-6074.patch";
```

Reply:
both locally and on github interface it looks gone 
https://github.com/NixOS/nixpkgs/pull/29226/files#diff-608de687ae1a5f9a6b1796f184eca73bL61

Reply:
In the `pkgs/os-specific/linux/kernel/patches.nix` file?
https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/linux/kernel/patches.nix#L61-L68

Reply:
either I am tired or you are checking out the wrong branch ? I do see the DCCP lines in pkgs/os-specific/linux/kernel/patches.nix prefixed with a '-' in https://patch-diff.githubusercontent.com/raw/NixOS/nixpkgs/pull/29226.diff

Reply:
Sorry not sure how I missed that

Reply:
no pb thanks for merging